packages/client/manifest.json

```diff
@@ -1825,19 +1883,19 @@
         "type": "select",
         "label": "Colors",
         "key": "palette",
-        "defaultValue": "Palette 1",
+        "defaultValue": "palette1",
         "options": [
-          "Custom",
-          "Palette 1",
-          "Palette 2",
-          "Palette 3",
-          "Palette 4",
-          "Palette 5",
-          "Palette 6",
-          "Palette 7",
-          "Palette 8",
-          "Palette 9",
-          "Palette 10"
+          { "label": "Custom", "value": "Custom" },
```

Comment:
A breaking change for existing charts with the palette value already specified. The values are currently stored as "Palette 1".

This is the same for all chart types

Reply:
Ah yeah that sucks, changed it back.